### PR TITLE
Set cookie to secure when SSL connection

### PR DESCRIPTION
**Pull request by davidhrbac**
Very small patch to implement secure cookie.

### CODE REVIEW

File: app/assets/javascripts/cookies_eu.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ var cookiesEu = {
   },
 
   setCookie: function() {
-    Cookies.set('cookie_eu_consented', true, { path: '/', expires: 365 });
+    Cookies.set('cookie_eu_consented', true, { path: '/', expires: 365, secure: location.protocol === 'https:' });
```
Comment (davidhrbac) [inline app/assets/javascripts/cookies_eu.js]: @gabskoro Are you OK with that?

**Comment by davidhrbac** [comment]
@gabskoro Here you are...

**Comment by gabskoro** [comment]
Looks fine now, will publish it soon, tnx :)

**Comment by davidhrbac** [comment]
Thanks fo quick action.

**Pull request merged**